### DISCUSSION TITLE
Is there any option available to split the token Site.Url?

### DISCUSSION THREAD

**Discussion by andikrueger** [microsoft-search/pnp-modern-search#2567] [General]
**Is your feature request related to a problem? Please describe.**

I'm working on a search based solution that lists all projects of a customer. These projects are stored in a list within a separate site collection. For each project, we create a new site collection. The url of this site collection https://customer.sharepoint.com/sites/project-id contains the project id.

We now plan to list some project details within the site collection. I would need to split the Site.Url to use this id within the search query.

**Describe the solution you'd like**
A way to split the Site.Url token to be used in the query

**Describe alternatives you've considered**
Am I missing any other option besides adding an URL column to my central list?

**Comment by FranckyC** [comment]
Hi @andikrueger, in the upcoming 4.8 release, we will add a query modifier extensibility option allowing you to update the query text the way you want before send it to the search engine. Stay tuned for the documentation and release.

**Reply by andikrueger** [reply]
Great news. Is there any ETA for 4.8?

**Reply by FranckyC** [reply]
This week normally
